scale-tests/deploy-dispatchers.py

```diff
@@ -167,30 +254,52 @@ def get_default_options(arguments: dict) -> dict:
     return options
 
 
+def get_quota_options(arguments: typing.Dict) -> typing.Dict:
+    """
+    Move the quota options from the command line arguments to a dict.
+    """
+    create_quotas = ast.literal_eval(arguments.get("--create-quotas", True))
+    if not create_quotas:
+        return {}
+
+    resources = ["cpus", "mem", "gpus", ]
+    targets = ["drivers", "executors", ]
+
+    quota_options = {}
+    for t in targets:
+        quota_options[t] = {}
+        for r in resources:
+            arg_key = "--quota-{}-{}".format(t, r)
+            if arg_key in arguments:
+                quota_options[t][r] = arguments[arg_key]
+
+    return quota_options
+
+
 if __name__ == "__main__":
     arguments = docopt(__doc__, version="deploy-dispatchers.py 0.0")
 
     options_file = arguments['--options-json']
     if options_file:
```

Comment:
We could also treat the options file in the same way as the config files for the infrastructure.